mindsdb/integrations/handlers/replicate_handler/Manual_QA.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to the MindsDB Manual QA Testing for Replicate Handler
+
+> **Please submit your PR in the following format after the underline below `Results` section. Don't forget to add an underline after adding your changes i.e., at the end of your `Results` section.**
+
+## Testing Replicate Handler 
+
+**1. CREATE MODEL**
+
+```sql
+CREATE MODEL stability
+PREDICT url
+USING
+    engine = 'replicate',
+    model_name= 'ai-forever/kandinsky-2',
+    version ='601eea49d49003e6ea75a11527209c4f510a93e2112c969d548fbb45b9c4f19f',
+    api_key = 'r8_BpOZN..............................';
+```
+
+**2. DESCRIBE PREDICTOR to see available parameters**
+
+```sql
+DESCRIBE PREDICTOR mindsdb.aiforever.features;

```

Comment:
I'm not sure I follow this, shouldn't it be mindsdb.stability?

Reply:
done